README.rst

```diff
@@ -0,0 +1,148 @@
+Chapel Protobuf
+===============
+This project is a Google Protocol Buffers implementation for Chapel.
+`Protocol Buffers`_ is a language-neutral, platform-neutral, extensible mechanism
+for serializing structured data. The protocol buffer language is a language for 
+specifying the schema for structured data. This schema is compiled into language
+specific bindings.
+
+This project comprises of two components:
+
+* Code generator: The `tool`_ is a compiler plugin to ``protoc``, the protocol
+  buffer compiler. It arguments the ``protoc`` compiler so that it knows how to
```

Comment:
Do you mean "augments" instead of "arguments"?

Reply:
So like "It provides arguments to the ``protoc`` compiler"?  As it stands now, "It arguments" is not a proper phrase - "arguments" is a noun and so is "it".  Two nouns shouldn't be next to each other in a sentence without a comma and since there is a relationship between the nouns rather than them being both part of a list or part of separate phrases, a comma is not appropriate.

Reply:
Yes, it sounds good, thanks for the explanation. Your grammar is really good!